tests/meta/backup/test_meta_backup.py

```diff
@@ -653,3 +664,32 @@ def test_decrypt_and_retrieve_spring_file_pdc_retrieval_failed(
 
     # THEN the decryption failure should be logged
     assert "Decryption failed" in caplog.text
+
+
+@pytest.mark.parametrize(
+    "copy_complete_exists",
+    [True, False],
+    ids=["copy_complete_exists", "copy_complete_does_not_exist"],
+)
+def test_create_copy_complete(
+    copy_complete_exists: bool, backup_api: BackupAPI, novaseq_x_flow_cell: FlowCellDirectoryData
+):
+    # GIVEN a flow cell that has been decrypted in flow_cell directory
+    flow_cell_dir: Path = novaseq_x_flow_cell.path
+    flow_cells: Path = flow_cell_dir.parent
+
+    # GIVEN the copy complete to be created
+    copy_complete_txt: str = "CopyComplete.txt"
+
+    # GIVEN or not a copy complete file exists
+    if copy_complete_exists:
+        novaseq_x_flow_cell.path.joinpath(copy_complete_txt).touch()
+    else:
+        novaseq_x_flow_cell.path.joinpath(copy_complete_txt).unlink(missing_ok=True)
```

Comment:
I don't understand this part: why should the path be touched if it already exists and deleted if not?

Reply:
Well, when we unpack the encrypted flow cell directory we can have two scenarios - either the file already exists, or it does not. Here I try to capture those two scenarios. I can make this into two tests, naming them test_touch_copy_complete_when_it_already_exists etc. but I just thought this was a neat way to do it.

Reply:
ohhh ok I got it